packages/pytest-simcore/src/pytest_simcore/helpers/faker_factories.py

```diff
@@ -437,3 +437,35 @@ def random_service_access_rights(
 
     data.update(**overrides)
     return data
+
+
+def random_itis_vip_available_download_item(
+    identifier: int,
+    fake: Faker = DEFAULT_FAKER,
+    features_functionality: str = "Posable",
+    **overrides,
+):
+    features_str = (
+        "{"
+        f"name: {fake.name()} Right Hand,"  # w/o spaces
+        f" version: V{fake.pyint()}.0,   "  # w/ x2 spaces
+        f"sex: Male, age: 8 years,"  # w/o spaces
+        f"date: {fake.date()}, "  # w/ x1 spaces
+        f"ethnicity: Caucasian, functionality: {features_functionality}  "
+        "}"
+    )
```

Comment:
maybe something like this is the same and nicer?
```suggestion
    features_dict = {
        "name": f"{fake.name()} Right Hand",  # w/o spaces
        " version": f"V{fake.pyint()}.0",  # w/ x2 spaces
        "sex": "Male, age: 8 years",  # w/o spaces
        "date": f"{fake.date()}",  # w/ x1 spaces
        "ethnicity": f"Caucasian, functionality: {features_functionality}  ",
    }
    features_str = json.dumps(features_dict)
```

Reply:
no, no... the vip API produces a json in a string :-)